docs/basic-features/script.md

```diff
@@ -64,28 +64,73 @@ With `next/script`, you decide when to load your third-party script by using the
 <Script src="https://connect.facebook.net/en_US/sdk.js" strategy="lazyOnload" />
 ```
 
-There are three different loading strategies that can be used:
+There are four different loading strategies that can be used:
 
-- `beforeInteractive`: Load before the page is interactive
+- `beforeInteractive`: Load scripts required by the entire site before the page is interactive
+- `beforePageRender`: Load scripts required by a particular page before the page is interactive
```

Comment:
Can we separate out `beforePageRender` mode into a separate PR? My understanding was that it would go through the RFC process first, and we need to start showing a warning soon since React 18 is imminent